### PR TITLE
fix: add aria-label and title attributes for the image and video buttons

### PR DESCRIPTION
**Pull request by julien**

### CODE REVIEW

File: src/assets/lang/en.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
 	"rows": "Rows",
 	"success": "Success",
 	"table": "Insert Table",
+	"video": "Insert Video",
```
Comment (wincent) [inline src/assets/lang/en.json]: I think we're supposed to add the keys to `src/assets/lang/language.json`, not this file.

File: src/components/buttons/button-embed-video.jsx [hunk 2]
```diff
@@ -46,10 +46,12 @@ class ButtonEmbedVideo extends React.Component {
 		} else {
 			return (
 				<button
+					arial-label={AlloyEditor.Strings.video}
```
Comment (wincent) [inline src/components/buttons/button-embed-video.jsx]: s/arial/aria/

I see what you did there...

Screen Shot 2019-05-27 at 16 01 51

File: src/assets/lang/en.json (hunk 1 above)
Comment (julien) [reply src/assets/lang/en.json]: Yes, you're right.

File: src/components/buttons/button-embed-video.jsx (hunk 2 above)
Comment (julien) [reply src/components/buttons/button-embed-video.jsx]: Nice good catch

**Pull request merged**